### PR TITLE
Improve README.md

### PR DESCRIPTION
**Pull request by easyaspi314**
@tonsky 
* I changed Do and Don't to Does and Doesn't 
* Netbeans and Dw CS6 don't work
* TextEdit does
* Fixed typos
* Fixed the unusual gap in the `all_ligatures.png` after the 6th group.
* Alphabetized. I failed twice…

Should we add that most Android apps work partially (at least on KitKat) by renaming FiraMono-Regular.otf to DroidSansMono.ttf and, using root access, copying it into /system/fonts? Standard ligatures (like `==` or `##`) work, but contextual alternates (like `0xF`) don't: it is forced to run in TrueType compatibility mode.

### CODE REVIEW

**Comment by easyaspi314** [comment]
@tonsky any thoughts?

File: README.md [hunk 1]
```diff
@@ -49,24 +49,25 @@ Work | Don’t work
...
+Probably works: **Smultron, Vico** |
 
 ### Browser support
 
-- IE 10+, Edge: enable with `font-feature-settings: "calt" 1;`
+- IE 10+ and Edge: enable with `font-feature-settings: "calt" 1;`
 - Firefox
 - Safari
-- Chromium-based browsers: enable with  `font-variant-ligatures: contextual;` or `text-rendering: optimizeLegibility` (see [571246](https://code.google.com/p/chromium/issues/detail?q=font-variant-ligatures&id=571246&thanks=571246&ts=1450553433&))
+- Chromium-based browsers (like Chrome, most Android browsers, and Opera): enable with  `font-variant-ligatures: contextual;` or `text-rendering: optimizeLegibility` (see [bug 571246](https://code.google.com/p/chromium/issues/detail?q=font-variant-ligatures&id=571246&thanks=571246&ts=1450553433&))
```
Comment (MadcapJake) [inline README.md]: most Android browsers? I don't think that Firefox and Dolphin are Chromium-based.

File: README.md [hunk 2]
```diff
@@ -16,31 +16,31 @@ Fira Code is an extension of the Fira Mono font containing a set of ligatures fo
 
 ### Terminal support
 
-Work | Don’t work
+Works | Doesn’t work
```
Comment (MadcapJake) [inline README.md]: This is grammatically confusing.  `Work` and `Don't Work` are the plural forms, `works` and `doesn't` are singular forms.

**Comment by MadcapJake** [comment]
Whether a given editor does or doesn't work should be addressed in its own issue/PR and discussed as to what it would take to get it working.  I don't think this is the kind of PR to be doing that in.

File: README.md [hunk 3]
```diff
@@ -49,24 +49,25 @@ Work | Don’t work
...
 **Scratch** |
 **TextAdept** (Linux, Mac) |
+**TextEdit** (Mac OS X text editor) |
 **TextMate 2** |
 **Visual Studio 2015** |
 **Visual Studio Code** ([instructions](https://github.com/tonsky/FiraCode/wiki/VS-Code-Instructions)) |
 **Xamarin Studio/Monodevelop** |
 **Xcode** (with [this plugin](https://github.com/robertvojta/LigatureXcodePlugin)) |
-Probably work: **Smultron, Vico** |
+Probably works: **Smultron, Vico** |
```
Comment (MadcapJake) [inline README.md]: Same grammar issue here, `work` is the appropriate form.

File: README.md (hunk 2 above)
Comment (easyaspi314) [inline README.md]: But you are addressing them singularly.

What about working/not working?

**Comment by tonsky** [comment]
Ok, I did some changes on top of the fresh file, merging this selectively is too painful. Thanks for the contribution!

**Comment by w-barath** [comment]
Firacode ligatures work fine in Netbeans  8.02 on Ubuntu 16.04, out of the box, no special config changes required other than selecting the Firacode font.  I am using openjdk version "1.8.0_91".

I also had it working on 14.04 before I upgraded to 16.04.  I think I had to make the following config changes for that to work:  

edit /usr/local/netbeans-8.0.2/etc/netbeans.conf

find the variable "netbeans_default_options" and change it to the following: 

```
netbeans_default_options="-J-client -J-Xss2m -J-Xms32m -J-XX:PermSize=32m -J-XX:MaxPermSize=200m
-J-Dapple.laf.useScreenMenuBar=true -J-Dapple.awt.graphics.UseQuartz=true
-J-Dsun.java2d.noddraw=true -J-Dswing.aatext=true -J-Dawt.useSystemAAFontSettings=on"
```

**Comment by easyaspi314** [comment]
Huh. I didn't get it on El Cap.

**Pull request closed without merging**